services/core-web/src/components/noticeOfWork/applications/review/activities/AccessRoads.js

```diff
@@ -1,69 +1,164 @@
+      render: (text, record) => (
+        <div title="Access Type">
+          <div className="inline-flex">
+            <input
+              name="activity_type_description"
+              type="text"
+              disabled={props.isViewMode}
+              value={text}
+              onChange={(e) => editActivity(e, record.index)}
+            />
+          </div>
+        </div>
+      ),
     },
     {
       title: "Length(km)",
       dataIndex: "length",
       key: "length",
-      render: (text) => <div title="Length(km)">{text}</div>,
+      render: (text, record) => (
+        <div title="Length(km)">
+          <div className="inline-flex">
+            <input
+              name="length"
+              type="text"
+              disabled={props.isViewMode}
+              value={text}
+              onChange={(e) => editActivity(e, record.index)}
+            />
+          </div>
+        </div>
+      ),
     },
     {
       title: "Disturbed Area (ha)",
-      dataIndex: "disturbedArea",
-      key: "disturbedArea",
-      render: (text) => <div title="Disturbed Area (ha)">{text}</div>,
+      dataIndex: "disturbed_area",
+      key: "disturbed_area",
+      render: (text, record) => (
+        <div title="Disturbed Area (ha)">
+          <div className="inline-flex">
+            <input
+              name="disturbed_area"
+              type="text"
+              disabled={props.isViewMode}
+              value={text}
+              onChange={(e) => editActivity(e, record.index)}
+            />
+          </div>
+        </div>
+      ),
     },
     {
       title: "Merchantable timber volume (m3)",
-      dataIndex: "timberVolume",
-      key: "timberVolume",
-      render: (text) => <div title="Merchantable timber volume (m3)">{text}</div>,
+      dataIndex: "timber_volume",
+      key: "timber_volume",
+      render: (text, record) => (
+        <div title="Merchantable timber volume (m3)">
+          <div className="inline-flex">
+            <input
+              name="timber_volume"
+              type="text"
+              disabled={props.isViewMode}
+              value={text}
+              onChange={(e) => editActivity(e, record.index)}
+            />
+          </div>
+        </div>
+      ),
     },
   ];
 
+  const removeColumn = {
+    dataIndex: "remove",
+    key: "remove",
+    render: (text, record) => (
+      <div name="remove" title="remove">
+        <Button
+          type="primary"
+          size="small"
+          onClick={() => props.removeRecord("exploration_access.details", record.index)}
+          ghost
+        >
+          <img name="remove" src={TRASHCAN} alt="Remove Activity" />
+        </Button>
+      </div>
+    ),
+  };
+
+  const columns = (isViewMode) =>
+    !isViewMode ? [...standardColumns, removeColumn] : standardColumns;
+
   const transformData = (activities) =>
-    activities.map((activity) => ({
-      type: activity.activity_type_description || Strings.EMPTY_FIELD,
```

Comment:
Have we move away from using Strings.EMPTY_FIELD?

Reply:
The problem is that if I use them then N/A will be passed to the backend on the form submit in the field.

Reply:
Thats strange behaviour. Is it worth a one-line comment?